package.json

```diff
@@ -4,30 +4,37 @@
         "dev": "vite",
         "build": "vite build",
         "lint": "npm run lint:eslint",
-        "lint:eslint": "eslint public/js resources/js postcss.config.js tailwind.config.js vite.config.js --cache",
+        "lint:eslint": "eslint public/js resources/js postcss.config.js tailwind.config.js vite.config.ts --cache",
         "fix": "npm run fix:eslint",
-        "fix:eslint": "eslint public/js resources/js postcss.config.js tailwind.config.js vite.config.js --fix",
+        "fix:eslint": "eslint public/js resources/js postcss.config.js tailwind.config.js vite.config.ts --fix",
         "analyze": "cross-env ANALYZE=true npm run prod",
         "apidoc": "npm run apidoc:connect",
         "apidoc:connect": "apidoc -i src/Connect/ -c docs/src/connect-api -t docs/src/connect-api/template -o docs/dist/api/connect"
     },
+    "dependencies": {
+        "alpinejs": "^3.0.6",
+        "js-cookie": "^3.0.1"
+    },
```

Comment:
Anything shipping directly to the browser I've moved into `"dependencies"`. This notably excludes tailwind because Tailwind is just a PostCSS plugin.

I am happy to undo this but this is within alignment of conventions with _package.json_ files.